Cargo.toml

```diff
@@ -7,4 +7,5 @@ edition = "2021"
 
 [dependencies]
 axum = "0.6"
+sqlx = { version = "0.6", features = [ "runtime-tokio-rustls", "sqlite" ] }
```

Comment:
We might want to add the `offline` feature, and do a `cargo sqlx prepare` so that we can build without the DB being available (e.g. in CI).

Reply:
Later we can add the migrate feature, or we can do it now.